src/Tribe/Ignored_Events.php

```diff
@@ -93,25 +93,19 @@ public function filter_updated_messages( $messages, $counts ) {
 			if ( 0 >= $count_ignored ) {
 				return $messages;
 			}
-
-			$messages[ Tribe__Events__Main::POSTTYPE ] = array(
-				'updated'   => _n( '%s Event updated.', '%s Events updated.', $counts['updated'], 'the-events-calendar' ),
-				'locked'    => _n( '%s Event not updated, somebody is editing it.', '%s Events not updated, somebody is editing them.', $counts['locked'], 'the-events-calendar' ),
-				'deleted'   => _n( '%s Event permanently deleted.', '%s Events permanently deleted.', $counts['deleted'], 'the-events-calendar' ),
-				'trashed'   => _n( '%s Event moved to the Trash', '%s Events moved to the Trash', $counts['trashed'], 'the-events-calendar' ),
-				'untrashed' => _n( '%s Event restored from the Trash.', '%s Events restored from the Trash.', $counts['untrashed'], 'the-events-calendar' ),
-			);
-
 			$counts['trashed'] -= $count_ignored;
 
-			// Resets the Variable used to count Trashed
-			$GLOBALS['bulk_counts'] = $counts;
+			$messages[ Tribe__Events__Main::POSTTYPE ] = $messages['post'];
 
-			if ( 0 < $counts['trashed'] ) {
-				$messages[ Tribe__Events__Main::POSTTYPE ]['trashed'] .= __( ' and ', 'the-events-calendar' );
+			// we are going to continue to use the language "posts" as that's what WordPress uses for custom post types in this messaging.
+			if ( 0 == $counts['trashed'] ) {
+				$messages[ Tribe__Events__Main::POSTTYPE ]['trashed'] = '%s ' . _n( 'post moved to Ignored.', 'posts moved to Ignored.', $count_ignored, 'the-events-calendar' );
+			}
+			else {
+				$GLOBALS['bulk_counts'] = $counts;
```

Comment:
nope, it only needs to be changed when there are posts that are both trashed and ignored. Otherwise, the `%s` that WordPress will replace is just the total, be it all ignored or all trashed